baseplate/lib/message_queue.py

```diff
@@ -140,6 +221,12 @@ def queue_tool() -> None:
         help="if creating the queue, what to set the maximum message size to",
     )
     parser.add_argument("queue_name", help="the name of the queue to consume")
+    parser.add_argument(
+        "--queue_type",
+        default="posix",
+        choices=["posix", "in_memory"],
```

Comment:
If you had a `QueueType` enum you could keep the choice in sync by replacing the list with:
```python
[qt.value for qt in QueueType]
```